definitions/modules/autopilot.yaml

```diff
@@ -1,134 +1,275 @@
 shared:
-  -
+  - # ASOBO_AUTOPILOT_Push_Autopilot_Template
     type: ToggleSwitch
     var_name: A:AUTOPILOT MASTER
     var_units: Bool
-    var_type: bool
     event_name: AP_MASTER
+
+  - # ASOBO_AUTOPILOT_Push_Disconnect_Template
+    type: NumSet
+    var_name: A:AUTOPILOT DISENGAGED
+    var_units: Bool
+    var_type: i32
+    event_name: AUTOPILOT_DISENGAGE_SET
   -
+    type: ToggleSwitch
+    var_name: A:AUTOPILOT DISENGAGED
+    var_units: Bool
+    event_name: AUTOPILOT_DISENGAGE_TOGGLE
+
+  - # ASOBO_AUTOPILOT_Push_Heading_Template
     type: ToggleSwitch
     var_name: A:AUTOPILOT HEADING LOCK
     var_units: Bool
-    var_type: bool
     event_name: AP_PANEL_HEADING_HOLD
+  -
+    type: ToggleSwitch
+    var_name: A:AUTOPILOT HEADING LOCK
+    var_units: Bool
+    event_name: AP_HDG_HOLD
+
+  - # ASOBO_AUTOPILOT_Knob_Heading_Template
+    type: NumSet
+    var_name: A:HEADING INDICATOR
+    var_units: Degrees
+    var_type: f64
+    event_name: HEADING_BUG_SET
   -
     type: NumSet
     var_name: A:AUTOPILOT HEADING LOCK DIR
     var_units: Degrees
     var_type: f64
     event_name: HEADING_BUG_SET
+  -
+    type: event
```

Comment:
You should not be including the increment/deincrement events for anything that has a NumSet to it. It has the same effect that H events has on radio definitions (the heading value will be thrown off by the increment event after it gets set)

Reply:
Events should be used as a **LAST RESORT** if you cannot find a NumSet/ToggleSwitch/Var